interface/modules/custom_modules/oe-form-templates/index.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ *
+ */
+
+use Google\Service\Books;
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Twig\TwigContainer;
+use OpenEMR\Modules\FormTemplates\Bootstrap;
+use OpenEMR\Modules\FormTemplates\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Twig\Extension\DebugExtension;
+
+require_once "../../../globals.php";
+require_once "../../../../library/registry.inc.php";
+
+// Control access
+if (!AclMain::aclCheckCore('patients', 'demo')) {
+    echo xlt('Not Authorized');
+    exit;
+}
+
+$templateDir = Bootstrap::getTemplatePath();
+$twigContainer = new TwigContainer($templateDir);
+$twig = $twigContainer->getTwig();
+$twig->addExtension(new DebugExtension());
+$twig->enableDebug();
+
+$request = Request::createFromGlobals();
+
+$r_controller = $request->get('controller', 'default');
+$r_action = $request->get('action', 'index');
+
+$routes = [
+    'default'       => 'OpenEMR\Modules\FormTemplates\Controller\ConfigurationController',
+    'configuration' => 'OpenEMR\Modules\FormTemplates\Controller\ConfigurationController',
+];
+
+if (array_key_exists($r_controller, $routes) == true) {
```

Comment:
I wonder if you could use PHP 8 attributes here to provide yourself some stronger typing here instead of using ReflectionClass for your frontend controller here (I'm referring to the design pattern, not a JS frontend).

Reply:
I will definitely look at this - I know using Reflection isn't the most optimized approach, it's kind of my default controller build right now, which I know can be improved.